### DISCUSSION TITLE
exiftool unexpected

### DISCUSSION THREAD

**Discussion by user-by** [tannerhelland/PhotoDemon#435] [General]
how to up date exiftool new version and fix "installed, but version unexpected" library status error?

**Comment by tannerhelland** [comment]
Hello @user-by .  I apologize for the delay in replying.  I have been on holiday for the past two weeks.

The ExifTool warning you describe is harmless.  You can update PhotoDemon's copy of ExifTool at any time, but I need to distinguish between "versions of ExifTool I have tested - and can guarantee work OK" and "versions of ExifTool I have *not* tested - which may have introduced new bugs."

If you encounter any issues with a newer copy of ExifTool, please let me know so I can fix them.  Otherwise, I typically wait for the next ExifTool release marked as "production-ready" before updating PhotoDemon's copy.

Hope that helps!

**Reply by user-by** [reply]
Thanks for reply, I am using your softwre since ver.7 and i always replacing with new version exiftool and i newer see any problem. Therefore i think this library mistake alerts  is not good for all users.  

Respects and greetings.

**Reply by user-by** [reply]
I saw exiftool update new in nightly-build, thanks for your concern, but i have a quescion can we expectes new update plugins to library?
